### ISSUE TITLE
What license?

### ISSUE THREAD

**Issue by TimSC** [mbalestrat/ECE-FYP-PneumoCheck#2]
This this code released under any particular license?

**Comment by mbalestrat** [comment]
No, there is no particular license. However it was completed as part of an undergraduate research project for Monash University.
